### DISCUSSION TITLE
分享一个自己开发的xray客户端

### DISCUSSION THREAD

**Discussion by ali322** [XTLS/Xray-core#832] [General]
qv2ray 停止维护了,一直没找到合适的客户端,只能自己用 tauri 撸了一个,因为只是自用所以功能比较简单,跨平台支持 mac 和 windows

![Snip20211129_1](https://user-images.githubusercontent.com/780693/143835619-61a1d2c9-a565-4809-aa0c-360581d6a487.png)

手头没windows 机器,先放 mac 的

**Comment by xtccc** [comment]
恕我直言 qv2ray 不维护也不是用不了， 等xray 更新的qv2ray不支持的时候再换吧

**Reply by ali322** [reply]
刚好在看 tauri 框架,所以拿来练手写了一个xray的客户端

**Reply by depfsh** [reply]
咋不开源出来

**Comment by ghost** [comment]
xflutter搞个tg群？

**Reply by ali322** [reply]
这个不是xflutter

**Comment by Rangtian** [comment]
赞

**Comment by zhao2077** [comment]
兄弟，readme 多写一些。我m1 机器。正缺客户端呢，这个如何编译。谢谢

**Reply by ali322** [reply]
m1可以直接用小火箭吧

**Comment by chenxiccc** [comment]
很有需要，怎么撤包了？

**Reply by ali322** [reply]
仓库地址 https://github.com/ali322/vess
